cmd/monaco/integrationtest/assert.go

```diff
@@ -119,12 +118,7 @@ func AssertAllConfigsAvailability(t *testing.T, fs afero.Fs, manifestPath string
 				continue
 			}
 
-			configParameters, errs := sort.Parameters(theConfig.Group, theConfig.Environment, theConfig.Coordinate, theConfig.Parameters)
-			testutils.FailTestOnAnyError(t, errs, "sorting of parameter values failed")
-
-			parameters = append(parameters, configParameters...)
```

Comment:
fun sidenote: this was pretty useless before.. we only ever happened the configParameters to it and directly used it to resolve parameter values...